README.md

```diff
@@ -323,3 +324,15 @@ In decending order of priority (top ones are more important):
       Maybe we can find a way, to wait for specific events to be triggered on the page. For example: Wait for all animations on the page to complete before taking the screenshot.
 
 If you find a bug or think some feature is missing, don't hesitate to submit an issue.
+
+## Building the Project
+
+Install [esy](https://esy.sh/) by running:
+```sh
+npm install -g esy
+```
+
+Install dependencies and build the project by running:
+```sh
+esy
+```
```

Comment:
Could you also add a section for running the tests?
They are run with `esy run test:yaml` and `esy run test:json`.

Reply:
Yep, added here: 56f1dff